### ISSUE TITLE
Address eventDate do's and don'ts

### ISSUE THREAD

**Issue by tucotuco** [tdwg/dwc-qa#107]
Issue borrowed from https://github.com/tdwg/dwc/issues/100.

**Comment by debpaul** [comment]
@tucotuco a great opportunity to use the data examples we have in our respective piles - to show what research can and can't be done if dates are in question. Also, to show how one might fix. And, to show what programming (scripting) can and can't fix in this regard. Might be a chance to connect with researchers (collectors) on this point as well so that future data is unambiguous in this respect.

**Comment by ekrimmel** [comment]
Comments from [DwC Hour #13](https://github.com/tdwg/dwc-qa/wiki/Webinars#chapter-13-the-problem-of-time-dealing-with-paleontological-and-zooarchaeological-specimens-in-darwin-core) suggest that it could be helpful to clarify that for specimens, eventDate should be the _collecting_ event. For extant organisms users may assume that the collecting event coincides with the time in which the organism was alive, but for paleo/zooarch objects this is not true.

**Comment by dagendresen** [comment]
If _dwc:eventDate_ should be strictly conforming to the "collecting event date" (domain = Occurrence?), then event-core datasets might need a separate (new?) _eventDate_ term with **domain = Event** ...?

**Comment by ekrimmel** [comment]
Ah! I didn't mean to suggest that DwC recommend restricting the use of eventDate. What I meant was that in our documentation here we could clarify that for paleo/zooarch occurrences eventDate is typically _not_ the same as the date when an organism was alive or in use.

**Comment by tucotuco** [comment]
Yet, I have counter examples in the zooarcheological specimens published to
VertNet, where they do use the chronometric date range as the eventDate so
that the meaning of when the organism occurred "in nature" is preserved.
There will need to be a clear resolution on this issue going forward. The
mix of concepts is not a good thing, and the use of only a collected or
observed date is limiting to the questions people want to investigate.

**Comment by debpaul** [comment]
seems clear that separate fields are needed where taxon is not extant to differentiate date collected/observed from time the specimen itself would have existed. Both are important to capture. John, since as you say "there will need to be a clear resolution on this issue going forward, " what's the next step to getting it addressed (getting a field/fields added).

**Comment by tucotuco** [comment]
The next step is for someone (it can't be me, by convention) to follow the procedures in section _3 Change Process_ in the TDWG Vocabulary Maintenance Specification [1]  for submitting requests for new terms in the Darwin Core, paying special attention to section _3.1 Justifications for Change_. 

[1] Vocabulary Maintenance Specification: https://github.com/tdwg/vocab/blob/master/vms/maintenance-specification.md

**Comment by hollyel** [comment]
So, are we suggesting that we need new terms so that we could, for example, maintain eventDate as the collecting/observation event and then a new term as the date the specimen was alive? Would the proposed chronometric extension then be an expansion on that new field for more thorough information with a paleo/zooarch context? (or the opposite, although I would observe it seems most records lean towards eventDate being the collecting/observation event, at least for paleo)

**Comment by tucotuco** [comment]
I think that is why it is a big community discussion. The meaning of
eventDate is pretty clear and entrenched, it just is not as useful as it
could be.

**Comment by dagendresen** [comment]
How about simply use the rule that if the dwc:eventDate is used as an attribute for an dwc:Occurrence then the eventDate has domain dwc:Occurrence (_i.e._ date describing the observation or sampling date, and not the time the organism was alive). And when eventDate used as an attribute for an event the dwc:eventDate simply has domain dwc:Event, and the type of event decides what type of date is described...? Minting new date terms for all types of events seems a never-ending story to me. But perhaps the time the organism was alive is specially important enough?

**Comment by tucotuco** [comment]
Still further consideration about best practice for eventDate can be found at https://github.com/tdwg/bdq/issues/86#issuecomment-402229992.

**Comment by DavidHepper** [comment]
I'm interested in date ranges of type 'number of days', where the event took place somewhere within a day range, but my complaint covers all ranges. 
It seems to me that the three interval examples at https://dwc.tdwg.org/list/#dwc_eventDate are misleading if the Start and End components of an eventDate are supposed to be inclusive. An example given is: 
`2007-11-13/15 (some time in the interval between the beginning of 13 November 2007 and before 15 November 2007)`
The "and before" implies that the interval ends at the _start_ of 15 November 2007 and so lasts 48 hours when I understand from ISO 8601-1 that 2007-11-13/15 ends at the _end_ of 15 November 2007 and so lasts 72 hours. That said, intervals are not well defined in the ISO standard, IMO.

**Comment by tucotuco** [comment]
Hi @DavidHepper. The changes to examples for terms that recommend adherence to ISO8601-1:2019 were ratified this year. The discussion leading to the correction of the examples following citations of the ISO standard can be found in Darwin Core issue https://github.com/tdwg/dwc/issues/558 about eventDate. 

The conclusion is that the example you cited is correct and that the interval "2007-11-13/15" ends at and does not include 15 November 2007 and so corresponds to 48 hours rather than 72 hours. Also note (because you say "lasts") that there is a distinction between an interval ("somewhen within") and a duration ("everywhen within").

**Comment by tucotuco** [comment]
Stated another way, because someone asked privately, "The interval includes everything between the instant designated by the value to the left of the '/' up to and not including the instant designated by the value to the right of the '/'.

**Comment by DavidHepper** [comment]
Thank you. I've never seen this before but do appreciate that it means the Start and End values can be subtracted to find the interval. "Weekdays" are usually defined as "Monday to Friday". The only thing similar is hotel bookings: arrive Friday and leave Sunday is only two nights. I suspect 90% of the range entries on NBN Atlas in the UK are technically wrong. I'll put on a hard hat and take it up with their overstretched administrators.

**Comment by tucotuco** [comment]
@DavidHepper Don't feel bad. I suspect 99% of all date data shared via Darwin Core are technically wrong for the simple reason that 2025-11-12 is an instant, not a day in ISO8601-1:2019.

**Comment by DavidHepper** [comment]
Thanks for the thought but I really think that's going too far. [This TDWG term definition](https://dwc.tdwg.org/list/#dwc_eventDate) of **eventDate** gives several examples showing that a single value is more than an instant, including: 

> **1809-02-12** (within the day 12 February 1809)

**Comment by tucotuco** [comment]
Yes, it makes thing extremely inconvenient that 1809-02-12 does not technically signify the whole day and that the example is actually in error given the recommendations.
